### PR TITLE
Update rails helper description for maintain_test_schema!

### PR DESCRIPTION
**Pull request by dcrec1**

### CODE REVIEW

File: lib/generators/rspec/install/templates/spec/rails_helper.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@
 # Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
 
 <% if RSpec::Rails::FeatureCheck.can_maintain_test_schema? -%>
-# Checks for pending migrations before tests are run.
+# Updates test schema before tests are run.
```
Comment (JonRowe) [inline lib/generators/rspec/install/templates/spec/rails_helper.rb]: If you want to change this I'd suggest `# Checks for pending migration and applies them before tests are run.` As the original text is currently clearer than your suggestion even if it's not the whole story.

Comment (dcrec1) [inline lib/generators/rspec/install/templates/spec/rails_helper.rb]: Ok, will change it.

Thanks

**Comment by dcrec1** [comment]
@JonRowe it's ok now?

**Comment by JonRowe** [comment]
(thanks)

**Pull request merged**